frontend/services/entities/mdm.ts

```diff
@@ -192,19 +193,17 @@ const mdmService = {
   // disk encryption too.
   updateAppleMdmSettings: (enableDiskEncryption: boolean, teamId?: number) => {
     const {
-      MDM_UPDATE_APPLE_SETTINGS: teamsEndpoint,
+      UPDATE_DISK_ENCRYPTION: teamsEndpoint,
       CONFIG: noTeamsEndpoint,
     } = endpoints;
     if (teamId === 0) {
       return sendRequest("PATCH", noTeamsEndpoint, {
         mdm: {
-          // TODO: API INTEGRATION: remove macos_settings when API change is merged in.
-          macos_settings: { enable_disk_encryption: enableDiskEncryption },
-          // enable_disk_encryption: enableDiskEncryption,
+          enable_disk_encryption: enableDiskEncryption,
         },
       });
     }
-    return sendRequest("PATCH", teamsEndpoint, {
+    return sendRequest("POST", teamsEndpoint, {
```

Comment:
Why did this change?

Reply:
Updating this call from using a deprecated endpoint to using the latest – see [here](https://github.com/fleetdm/fleet/blob/f91266a87e6d749de44b534cab9297ecf05bd87c/server/service/handler.go#L734)

Reply:
Actually looks like we can unify logic for which endpoint to hit here, as the new endpoint supports a nil team ID for editing no-team config, vs. needing to split between a team-specific endpoint and a no-team-specific endpoint. Does that match what you're seeing?

Reply:
Yes, nice!